index.js

```diff
@@ -143,18 +140,8 @@ class Hyperdiscovery extends EventEmitter {
         if (!stream.destroyed) stream.destroy(err)
       }
 
-      self._replicatingFeeds.set(dkeyStr, feed)
-
-      if (!feed || !feed.isSwarming) {
```

Comment:
I think we should keep the guard for `!feed || !feed.isSwarming` in for weird use cases like dat-store.

Reply:
`feed` is checked for being null in like 138 and it seems like isSwarming is not needed, but I'd have to look at it again more closely

Reply:
Also wondering about the try catch, is that needed?

Reply:
Whoops, didn't see that in the git diff! I'm not sure what the try-catch could be for. 🤔 It was added [in this commit](https://github.com/datproject/hyperdiscovery/commit/9cece8583ca0748a238a3873da2d2bd80b60f8d4) and presumably has something to do with handling multiple feeds. Any idea why that might be @joehand ?